docs/calculations.mdx

```diff
@@ -132,32 +132,32 @@ Native product carousel
-| `creative_time_in_view_seconds`      | Time that the creative was visible on the device                                            |
+| Field                                      | Description                                                                                 |
+| ------------------------------------------ | ------------------------------------------------------------------------------------------- |
+| `impressions`                              | The number of impressions counted (required for all channels other than DOOH)               |
+| `views`                                    | The number of views counted                                                                 |
+| `plays`                                    | The number of plays (required for DOOH)                                                     |
+| `utc_datetime`                             | Date and time, in UTC, when impressions were delivered                                      |
+| `country`                                  | The country where the impression was delivered                                              |
+| `region`                                   | The region of the country where the impression was delivered                                |
+| `channel`                                  | See [channels](#channels)                                                                   |
+| `device_type`                              | See [device types](#device-types)                                                           |
+| `network_type`                             | See [network types](#network-types)                                                         |
+| `property`                                 | See [property](#property)                                                                   |
+| `creative_ad_format`                       | Either a basic or vendor-provided ad format                                                 |
+| `creative_ad_platforms`                    | Ad serving, verification, and measurement ad platforms wrapping or embedded in the creative |
+| `creative_total_image_data_transfer_bytes` | Total bytes for the data transfer for the image itself , ideally measured by the CDN        |
+| `creative_image_sizes`                     | Array of image sizes included in the creative                                               |
+| `creative_total_audio_data_transfer_bytes` | Total bytes for the data transfer for the core audio asset, ideally measured by the CDN     |
+| `creative_audio_duration_seconds`          | Audio duration in seconds                                                                   |
+| `creative_total_video_data_transfer_bytes` | Total bytes for the data transfer for the core video asset, ideally measured by the CDN     |
+| `creative_video_vast_bytes`                | Size of VAST/VPAID wrapper for the video in bytes                                           |
```

Comment:
should this be average? I guess same for the other metrics - should we clarify?

Reply:
Yeah, this is fair point. Will add into the descriptions